debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/LegacyV1SourceInfoTest.java

```diff
@@ -158,6 +158,23 @@ public void shouldRecoverSourceInfoFromOffsetWithNonZeroBinlogCoordinatesAndNonZ
 
     @Test
     public void shouldRecoverSourceInfoFromOffsetWithFilterData() {
+        final String databaseWhitelist = "a,b";
```

Comment:
Should use new names?

Reply:
not in tests, like we agreed in the discussion on what we are going to change. that will blow up the PR a lot!

Reply:
Hum, ok. Can you create a follow-up Jira issue for this then. Thx.

Reply:
I logged https://issues.redhat.com/browse/DBZ-2433 for this